src/css/support.js

```diff
@@ -46,61 +46,57 @@ define([
 		documentElement.removeChild( container );
 	}
 
-	// Support: node.js jsdom
-	// Don't assume that getComputedStyle is a property of the global object
```

Comment:
I'd like to get input from more people here as I'm a little afraid of this change. cc @jquery/core